foundations/foundation-config/src/main/java/org/apache/servicecomb/config/archaius/sources/MicroserviceConfigLoader.java

```diff
@@ -38,14 +40,13 @@
   private static final String DEFAULT_CONFIG_FILE_NAME = "microservice.yaml";
 
   public MicroserviceConfigLoader() {
-    setOrderKey("cse-config-order");
   }
 
   public void loadAndSort() {
     try {
       String configFileFromClasspath =
-          System.getProperty("cse.configurationSource.defaultFileName") == null ? DEFAULT_CONFIG_FILE_NAME
-              : System.getProperty("cse.configurationSource.defaultFileName");
+          System.getProperty(DEFAULT_FILE_NAME) == null ? DEFAULT_CONFIG_FILE_NAME
```

Comment:
not compatible?
because we only copy config items in configuration source, not in system.property?

Reply:
Yes. Intended, we have never mention this in docs. And I add new docs for this change. See java-chassis-doc PR.

Reply:
yes, nobody use this feature, we can changed it directly

Reply:
Throw an error for this wrong configuration and force users change to correct key.